python/ml4ir/base/tests/test_robust_scaler_transform.py

```diff
@@ -33,7 +33,7 @@ def test_reciprocal_rank_negative_scores(self):
 
         self.assertTrue(np.isclose(actual_reciprocal_ranks, expected_reciprocal_ranks).all())
 
-    def test_reciprocal_rank_zeros(self):
+    def test_robust_scaler_zeros(self):
```

Comment:
what is the reason behind renaming ?

Reply:
I think the naming was just wrong earlier. Probably a copy-paste error. Can you confirm @mohazahran ?